src/test/regress/expected/multi_repair_shards.out

```diff
@@ -1,4 +1,6 @@
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 820000;
+SELECT groupid AS worker_2_group FROM pg_dist_node WHERE nodeport=:worker_2_port \gset
+SELECT groupid AS worker_1_group FROM pg_dist_node WHERE nodeport=:worker_1_port \gset
```

Comment:
I'm planning on addressing this, and adding `groupid` to master_add_node(), in the PR where we add `noderole`.